### PR TITLE
APISUPPORT-914-versioning-and-beta

### PR DESCRIPTION
**Pull request by AureliuszBrussy**
Changing info about versioning and beta version to match the guidelines to our first ADR

### CODE REVIEW

**Review by hjagodzinski — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -266,6 +254,13 @@ Content-Type: application/vnd.allegro.beta.v1+json
 Accept: application/vnd.allegro.beta.v1+json
 ```
 
+Rules for new beta resources:
+* within a given version, beta resources can be subject to modifications. As a result, we do not recommend the production use of beta resources,
```
Comment (hjagodzinski) [inline README.md:258]: I would like to rephrase this point to make sure I understand it correctly. Assuming for instance that once a new resource `application/vnd.allegro.beta.v1+json` is released can it be changed without introducing a new version `application/vnd.allegro.beta.v2+json`?

Comment (AureliuszBrussy) [reply README.md:258]: With the beta version, teams can make changes that do not break beta compatibility, but with breakers, we recommend implementing the new beta.v2 (within reason). With the beta version, we are not and never have been so strict - and we feel that this should not change. The case is of course different if the team has already decided that resource is coming out of beta.

Comment (hjagodzinski) [reply README.md:258]: My understanding is that. As long as backward compatibility is preserved the recommendation is to stick to the current beta version. As soon as a breaking change is introduced it's recommended to release a new beta version. Finally, when a resource is production-ready the version is set to public. Is it what you have in mind?

Comment (AureliuszBrussy) [reply README.md:258]: Exactly.

Comment (hjagodzinski) [reply README.md:258]: That's reasonable. Thanks 👍

**Pull request merged**